lib/tasks/npm-task.js

```diff
@@ -13,47 +20,86 @@ class NpmTask extends Task {
     // The command to run: can be 'install' or 'uninstall'
     this.command = '';
 
-    this.npm = this.npm || require('../utilities/npm');
+    this.versionConstraints = '3 || 4';
+  }
+
+  npm(args) {
+    logger.info('npm: %j', args);
```

Comment:
args is an array, yes. %j uses JSON formatting, while the default toString of an array can hide information